$NetBSD: patch-ae,v 1.2 2008/04/08 01:44:47 bjs Exp $

I'm not quite sure what's going on here, but I don't think res
should be a pointer (quells a warning, has no ill effects that
I can detect).

--- libjack/intclient.c.orig	2008-03-13 05:05:16.000000000 -0400
+++ libjack/intclient.c
@@ -167,7 +167,7 @@ jack_internal_client_load (jack_client_t
 {
 	va_list ap;
     va_start(ap, status);
-    jack_intclient_t* res = jack_internal_client_load_aux(client, client_name, options, status, ap);
+    jack_intclient_t res = jack_internal_client_load_aux(client, client_name, options, status, ap);
     va_end(ap);
     return res;
 }
